packages/cms/src/components/sections/Related.jsx

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from "react";
+import PropTypes from "prop-types";
+import {hot} from "react-hot-loader/root";
+import {connect} from "react-redux";
+import linkify from "../../utils/linkify";
+import Tile from "./components/Tile";
+
+import "./Related.css";
+
+class Related extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tiles: []
+    };
+  }
+
+  componentDidMount() {
+    const {profiles, locale} = this.props;
+    const {router} = this.context;
+
+    // make sure we got profiles
+    if (profiles && profiles.length) {
+      // generate tiles from profiles array
+      const tiles = profiles.map(profile => ({
+        title: profile.length === 1 ? profile[0].name : `${profile[0].name} / ${profile[1].name}`,
+        link: linkify(router, profile, locale),
+        images: profile.map(d => ({src: `/api/image?slug=${d.slug}&id=${d.id}&size=thumb`}))
+      }));
+
+      this.setState({tiles});
+    }
+
+    return null;
+  }
```

Comment:
@jhmullen a word of the wise: because nothing renders on initial mount, and you/James is formatting the data in componentDidMount, that means that these tiles aren't rendered in the server-side payload... which defeats the whole SEO purpose of these links! I'm refactoring this now to use the Tile component that I made for the search, so no worries about fixing this (I'll take care of it)